doc/nginx-setup.md

```diff
@@ -18,7 +18,7 @@ block
 
```

Comment:
@ostapneko Yep, I don't love that script either, just a thought. The steps are pretty clear anyway. :+1: